modules/mono/glue/GodotSharp/GodotSharp/Core/GD.cs

```diff
@@ -190,7 +190,7 @@ public static Resource Load(string path)
         ///
         /// Note: Errors printed this way will not pause project execution.
         /// To print an error message and pause project execution in debug builds,
-        /// use [code]assert(false, "test error")[/code] instead.
+        /// use <c>Debug.Assert(false, "test error")</c> instead.
```

Comment:
This documentation is still technically incorrect since the project execution won't pause, however `Debug.Assert` is probably the closest thing C# has to GDScript's `assert`. Maybe this note should just be removed in the C# documentation.

Reply:
I think we should remove these two lines. The first line of the note is fine though.